instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/semconv/http/HttpServerRoute.java

```diff
@@ -97,7 +97,7 @@ public static <T, U> void update(
       HttpServerRouteBiGetter<T, U> httpRouteGetter,
       T arg1,
       U arg2) {
-    Span serverSpan = LocalRootSpan.fromContextOrNull(context);
+    Span serverSpan = LocalRootServerSpan.fromContextOrNull(context);
```

Comment:
could we store the server span in `HttpRouteState` instead?

Reply:
I changed it to `HttpRouteState`, the downside with this is that bridging is a bit more complicated.